### PR TITLE
build: get sample apps compiling after flutter update

### PR DESCRIPTION
**Pull request by levibostian**
When trying to get the flutter sample app compiling on iOS, I got some errors during compilation. These updates were what I needed to change to get the app to compile again.

### CODE REVIEW

File: apps/amiapp_flutter/lib/src/color_schemes.g.dart
Comment (levibostian) [inline apps/amiapp_flutter/lib/src/color_schemes.g.dart:1]: The colors I picked were somewhat random. I chose white colors for the light theme and dark colors for the dark theme.

File: apps/amiapp_flutter/lib/src/color_schemes.g.dart [hunk 1]
```diff
@@ -27,7 +27,6 @@ const lightColorScheme = ColorScheme(
   onErrorContainer: Color(0xFF410002),
   surface: Color(0xFFFFFBFF),
   onSurface: Color(0xFF1B1B1F),
-  surfaceContainerHighest: Color(0xFFE3E1EC),
```
Comment (levibostian) [inline apps/amiapp_flutter/lib/src/color_schemes.g.dart:30]: Compiler said that this property is no longer used.

File: apps/amiapp_flutter/lib/src/color_schemes.g.dart [hunk 2]
```diff
@@ -69,4 +69,6 @@ const darkColorScheme = ColorScheme(
   surfaceTint: Color(0xFFBCC2FF),
   outlineVariant: Color(0xFF46464F),
   scrim: Color(0xFF000000),
+    background: Color(0xFF000000),
+    onBackground: Color(0xFFC7C5D0)
```
Comment (mrehan27) [inline apps/amiapp_flutter/lib/src/color_schemes.g.dart:73]: I think the file needs a few indentation fixes

File: apps/amiapp_flutter/pubspec.lock [hunk 3]
```diff
@@ -678,4 +678,4 @@ packages:
     version: "3.1.2"
 sdks:
   dart: ">=3.3.0 <4.0.0"
-  flutter: ">=3.18.0-18.0.pre.54"
+  flutter: ">=3.7.0"
```
Comment (mrehan27) [inline apps/amiapp_flutter/pubspec.lock:681]: I see this and a few other changes for downgrading versions. Could this be the reason for getting those errors? Can you try updating Flutter version and see if that helps?

If that fixes the problem, would be happy to discuss internally on how can we make this consistent across our team and have clear recommendations for versions. But I want to make sure we understand the root cause and we have a reason to downgrade these versions.

**Review by mrehan27 — COMMENTED** [review]
My only concern is around downgrading versions. Once that discussion is resolved, the PR looks good to merge.

File: apps/amiapp_flutter/pubspec.lock (hunk 3 above)
Comment (levibostian) [reply apps/amiapp_flutter/pubspec.lock:681]: Oh, that is weird. I am not sure why this happened? 

Let me try an upgrade first and see what happens.

Comment (levibostian) [reply apps/amiapp_flutter/pubspec.lock]: Not sure what caused these lock file changes but I seem to have fixed it. 

The iOS app is compiling successfully with the code in this PR. Ready for re-review.

**Review by mrehan27 — APPROVED** [review]
Looks good 👍🏻

**Comment by levibostian** [comment]
I think that my version of flutter was out-of-date or missconfigured on my machine. 

I upgraded to the latest version of Flutter and now the app runs on `main` with no modifications. Closing PR, ignore it.

**Pull request closed without merging**